src/app/public/testing/country-field/country-field-fixture.spec.ts

```diff
@@ -0,0 +1,209 @@
+import {
+  fakeAsync,
+  ComponentFixture,
+  TestBed,
+  tick
+} from '@angular/core/testing';
+
+import {
+  Component
+} from '@angular/core';
+
+import {
+  expect,
+  SkyAppTestUtility
+} from '@skyux-sdk/testing';
+
+import {
+  SkyCountryFieldCountry
+} from '@skyux/lookup';
+
+import {
+  SkyCountryFieldTestingModule
+} from './country-field-testing.module';
+
+import {
+  SkyCountryFieldFixture
+} from './country-field-fixture';
+
+const DATA_SKY_ID = 'test-country-field';
+const COUNTRY: SkyCountryFieldCountry = {
+  name: 'United States',
+  iso2: 'us'
+};
+
+//#region Test component
+@Component({
+  selector: 'country-field-test',
+  template: `
+  <sky-country-field
+    data-sky-id="${DATA_SKY_ID}"
+    formControlName="countryControl"
+    [autocompleteAttribute]="autocompleteAttribute"
+    [disabled]="disabled"
+    [hideSelectedCountryFlag]="hideSelectedCountryFlag"
+    (selectedCountryChange)="selectedCountryChange($event)"
+  >
+  </sky-country-field>
+  `
+})
+class CountryFieldTestComponent {
+  public autocompleteAttribute: string;
+  public disabled: boolean;
+  public hideSelectedCountryFlag: boolean;
+
+  public selectedCountryChange(query: string) { }
+}
+//#endregion Test component
+
+describe('Country field fixture', () => {
+  let fixture: ComponentFixture<CountryFieldTestComponent>;
+  let testComponent: CountryFieldTestComponent;
+  let countryFieldFixture: SkyCountryFieldFixture;
+
+  //#region helpers
+
+  function detectChangesFakeAsync(): void {
+    fixture.detectChanges();
+    tick();
+  }
+
+  //#endregion
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [
+        CountryFieldTestComponent
+      ],
+      imports: [
+        SkyCountryFieldTestingModule
+      ]
+    });
+
+    fixture = TestBed.createComponent(
+      CountryFieldTestComponent
+    );
+    testComponent = fixture.componentInstance;
+    fixture.detectChanges();
+    countryFieldFixture = new SkyCountryFieldFixture(fixture, DATA_SKY_ID);
+  });
+
+  it('should expose the expected defaults', fakeAsync(async () => {
```

Comment:
Do all of these tests need to be `async`? We probably also shouldn't be mixing `fakeAsync` and `async`.